chapters/ch10/1.setup-the-movie-db.ipynb

```diff
@@ -67,27 +65,28 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "GET https://github.com/ai-powered-search/tmdb/raw/main/judgments.tgz\n",
-      "GET https://github.com/ai-powered-search/tmdb/raw/main/movies.tgz\n",
+      "Wiping \"tmdb\" collection\n",
```

Comment:
This looks like OpenSearch output. Please ensure that Solr output is the default.